form-correction.html

```diff
@@ -960,9 +960,6 @@ <h4>Integrity Pledge
       }
     });
   </script>
-
-
```

Comment:
@Albert-Jokelin there are so many redundant changes like this please revert all.